hudi-io/src/main/java/org/apache/hudi/storage/HoodieStorage.java

```diff
@@ -332,6 +350,18 @@ public boolean createNewFile(StoragePath path) throws IOException {
     }
   }
 
+  /**
+   * Opens an SeekableDataInputStream at the indicated path with seeks supported.
+   *
+   * @param path the file to open.
+   * @return the InputStream to read from.
+   * @throws IOException IO error.
+   */
+  @PublicAPIMethod(maturity = ApiMaturityLevel.EVOLVING)
+  public SeekableDataInputStream openSeekable(StoragePath path) throws IOException {
```

Comment:
minor: should put this next to the other openSeekable def

Reply:
I intentionally put the APIs that do not need to be implemented at the end of the class.

Reply:
ok

Reply:
should it be final then?

Reply:
I think this one can still be overridden if the particular storage implementation provides optimization on this one.